docs/kb/semgrep-ci/scan-monorepo-in-parts.md

```diff
@@ -19,7 +19,7 @@ As such, it can be helpful to scan a monorepo in parts for multiple reasons:
 
 When scanning a repo with Semgrep in CI, the base command is `semgrep ci`. To understand this default setup for your source code manager (SCM) and CI provider, see [Getting started with Semgrep in continuous integration (CI)](/deployment/add-semgrep-to-ci).
```

Comment:
Is this meant to be `repo`, or `monorepo`?

Reply:
It's meant to be repo since we are just talking about running a Semgrep scan in general here, not specific to monorepos.